src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+logo.addEventListener('click', () => {
+  handleClick();
+});
+
+const handleClick = () => {
+  const promise1 = new Promise((resolve) => {
+    resolve('Promise was resolved!');
+  });
+
+  promise1.then(handleSuccess).catch(handleError);
+};
+
+const createRejectingPromise = (message) => {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      reject(new Error(message));
+    }, 3000);
+  });
+};
+
+const promise2 = createRejectingPromise('Promise was rejected!');
+
+const messageDiv = document.createElement('div');
+
+document.body.appendChild(messageDiv);
+
+const handleSuccess = (message) => {
+  messageDiv.className = 'message';
+  messageDiv.textContent = message;
+};
+
+const handleError = (errorMessage) => {
+  messageDiv.className = 'message error-message';
+  messageDiv.textContent = errorMessage;
```

Comment:
In this line, you're setting the textContent of the message div to the error object. However, the error object is not a string, so it will be converted to a string, which results in the text '[object Object]' being displayed. To display the error message, you should use `errorMessage.message` instead of `errorMessage`.